bin/server.js

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/env node
+import Debug from 'debug';
+import base from 'taskcluster-base';
+import api from '../lib/api';
+import path from 'path';
+import common from '../lib/common';
+import Promise from 'promise';
+import _ from 'lodash';
+
+let debug = Debug('secrets:server');
+
+/** Launch server */
+let launch = async function(profile) {
+  debug("Launching with profile: %s", profile);
+  var cfg = common.loadConfig(profile);
+
+  try {
+    var statsDrain = common.buildInfluxStatsDrain(
+      cfg.get('influx:connectionString'),
+      cfg.get('influx:maxDelay'),
+      cfg.get('influx:maxPendingPoints')
+    );
+  } catch(e) {
+    debug("Missing influx_connectionStraing: stats collection disabled.");
+    var statsDrain = common.stdoutStatsDrain;
+  }
+
+  // Start monitoring the process
+  base.stats.startProcessUsageReporting({
+    drain:      statsDrain,
+    component:  cfg.get('taskclusterSecrets:statsComponent'),
+    process:    'server'
+  });
+
+  let validator = await common.buildValidator(cfg);
+
+   let entity = base.Entity.configure({
+    version:          1,
+    signEntities:     cfg.get('azure:signEntities'),
```

Comment:
Don't make this configurable... What's the point, for testing just hardcode a dummy secret into the config file..

That way you also get this tested :)